Formula/t/talisman.rb

```diff
@@ -24,6 +24,10 @@ class Talisman < Formula
   depends_on "go" => :build
 
   def install
+    # go: updates to go.mod needed; to update it:
+    # go mod tidy
+    system "go", "mod", "tidy"
```

Comment:
need to upstream the change

Reply:
I ran it locally and compared the go.mod file before/after running `go mod tidy`.

This gives me:
```
--- go.mod	2025-04-18 20:38:58
+++ go.mod_back	2025-04-18 20:38:41
@@ -34,6 +34,4 @@
 	gopkg.in/yaml.v3 v3.0.0 // indirect
 )

-go 1.23.0
-
-toolchain go1.24.1
+go 1.18
```

I unsure about what to exactly upstream? (I have no real experience in go packaging so I'm trying to understand the rationale).

Reply:
yeah, the issue is with go 1.18 identifier, otherwise it would need to download the go toolchain to match with the local golang version